recipes/nb_conda_kernels/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.0.3" %}
+{% set gh_org = "Anaconda-Platform" %}
+{% set gh_repo = "nb_conda_kernels" %}
+
+package:
+  name: {{ gh_repo }}
+  version: {{ version }}
+
+source:
+  fn: {{ gh_repo }}-{{ version }}.tar.gz
+  url: https://github.com/{{ gh_org }}/{{ gh_repo }}/archive/{{ version }}.tar.gz
+  sha256: 6cdc08c3337c06e5cffacce150dbd10f87334489768e4c8804e045d28f65532a
+
+build:
+  number: 1
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - notebook >=4.2.0
+    - python
+
+test:
+  requires:
+    - coverage
+    - flake8
+    - nodejs
+    - nose
+    - notebook >=4.2.0
+    - python
+    - requests
+    - mock  # [py2k]
```

Comment:
Still figuring out how to do the stdout in postlink, but will reset build...

As for mock: will have to research... not sure when it was added to
stdlib...

On 13:56, Sat, Jun 11, 2016 Filipe <notifications@github.com> wrote:

> In recipes/nb_conda_kernels/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/702#discussion_r66709436>
> :
>
> > +    - setuptools
> > +
> > +  run:
> > +    - notebook >=4.2.0
> > +    - python
> > +
> > +test:
> > +  requires:
> > +    - coverage
> > +    - flake8
> > +    - nodejs
> > +    - nose
> > +    - notebook >=4.2.0
> > +    - python
> > +    - requests
> > +    - mock  # [py2k]
>
> I guess this should be [<py35], no?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/702/files/2040c9af1c634d31bf56601a8c123ec33bdec77a#r66709436>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AACxRKil06UX7JAuHgPhIYQgETglIw4Dks5qKvbUgaJpZM4Is3kX>
> .
>